sphinx/source/docs/releases/0.12.4.rst

```diff
@@ -125,6 +125,13 @@ BokehJS has migrated to using ``.tsx`` templates. Provisional support for
 including ``.eco`` templates has been removed from both BokehJS and the
 Bokeh Python library.
 
+Python 3.3 No Longer Supported
```

Comment:
@mattpap this should probably be moved to 0.10

In the 0.12.4 relnotes I could note instead that the use of b64encode means np arrays no longer work?